app/assets/javascripts/admin/addon/templates/emojis-index.hbs

```diff
@@ -0,0 +1,70 @@
+<AdminPageSubheader @titleLabel="admin.emoji.title">
+  <:actions as |actions|>
+    <actions.Primary @route="adminEmojis.new" @label="admin.emoji.new" />
+  </:actions>
+</AdminPageSubheader>
+
+<div class="form-horizontal">
+  <div class="inline-form">
+    <label class="label">Show</label>
+    <ComboBox
+      @value={{this.filter}}
+      @content={{this.sortingGroups}}
+      @nameProperty={{null}}
+      @valueProperty={{null}}
+      @onChange={{action "filterGroups"}}
+    />
+  </div>
+</div>
+
+{{#if this.sortedEmojis}}
```

Comment:
we show no message is no results? @ellaestigoy

Reply:
we should show have empty state message here 👍🏼